lawnchair/src/app/lawnchair/ui/preferences/IconPackPreferences.kt

```diff
@@ -135,15 +135,21 @@ fun IconPackPreferences() {
                 modifier = Modifier.padding(bottom = 8.dp),
             )
             PreferenceGroup {
-                val themedIconsAvailable = LocalContext.current.packageManager
-                    .getThemedIconPacksInstalled(LocalContext.current)
+                val themedIconPack = LocalContext.current.packageManager.getThemedIconPacksInstalled(LocalContext.current)
+                val themedIconsAvailable = themedIconPack
                     .any { LocalContext.current.packageManager.isPackageInstalled(it) }
                     || LocalContext.current.packageManager
                     .isPackageInstalled(Constants.LAWNICONS_PACKAGE_NAME)
                 ListPreference(
                     enabled = themedIconsAvailable,
```

Comment:
I'd suggest disabling the preference when the icon pack doesn't support themed icons:
```suggestion
                    enabled = themedIconsAvailable && supportsThemedIconPack,
```